src/main/pages/che-7/administration-guide/con_persistent-volumes-backups.adoc

```diff
@@ -9,24 +9,29 @@ summary:
 ---
 
 :page-liquid:
-:parent-context-of-persistent-volumes-backups: {context}
 
 [id="persistent-volumes-backups_{context}"]
 = Persistent Volumes backups
 
-Persistent Volumes (PVs) store the {prod-short} workspace data like a source code of a repository. 
+Persistent Volumes (PVs) store the {prod-short} workspace data in the form of a source code in a repository. 
```

Comment:
Does it really do that?
I it drew a comparison in this sentence,  as in `it stores the data like repository stores the code`

Reply:
@rkratky the statement in the fix misleading

Reply:
Thanks both. Fixing.

Reply:
@ibuziuk, would this be correct?

_...workspace data similarly to how source code is stored in a repository._

Reply:
well, I'm not sure I get what is the problem with the original statement - PV stores workspace data - source code, scripts, configuration files etc.